Sources/OpenCombineFoundation/Helpers/Violations.swift

```diff
@@ -0,0 +1,17 @@
+//
+//  Violations.swift
+//  
+//
+//  Created by Sergej Jaskiewicz on 13.12.2019.
+//
+
+import OpenCombine
+
+extension Subscribers.Demand {
+    internal func assertNonZero(file: StaticString = #file,
+                                line: UInt = #line) {
+        if self == .none {
```

Comment:
```suggestion
        guard self != .none else {
```

guarantees that negative case is actually handled this way

Reply:
I have this weird preference to not use `guard` with negation. In my opinion, it makes code harder to read.